### PR TITLE
Typos from ccache history

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -5308,6 +5310,7 @@ doesent'->doesn't
 doesent->doesn't
 doesing->doing, does, does in, dosing, dozing,
 doesn;t->doesn't
+doesnexist->doesntexist
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: This should correct to `doesn't exist`, but I can't remember if we support multiple words in the correction currently.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -6151,6 +6154,7 @@ excempt->exempt
 excepetion->exception
 excepions->exceptions
 exceptation->expectation
+exceptins->exceptions
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also suggest excepting

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -14195,6 +14203,7 @@ statisfies->satisfies
 statisfy->satisfy
 statisfying->satisfying
 statisitics->statistics
+statitics->statistics
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the singular version too please.

**Comment by peternewman** [comment]
@Gelma can we start with the ones that have already been reviewed like this one?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**